knexfile.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+
+  development: {
```

Comment:
Testing and development are pretty similar words and where do you find the difference in which will be used in public? I would recommend making one of them into a "production".